2023-10-13/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+)
+
+func appendEl(s string) string {
```

Comment:
To append is basically to add an element onto another. It can be to add an element to an array or more specifically like in this case, adding string el to Jo. The append function doesn't exist natively in go but is defined here to add variable "s" to string "el".

Reply:
`append()` actually exists, but for the purpose of appending an element to the end of a slice
https://pkg.go.dev/builtin#append